tests/queries/0_stateless/00305_http_and_readonly.reference

```diff
@@ -1,19 +1,19 @@
-name               value   changed
+ name               value   changed
```

Comment:
This change looks suspicious, but probably it's ok for `PrettySpace`.
Hopefully it won't brake somebody's parser.

Reply:
Yes, this whitespace is to make code simpler :)